plugins/modules/vmware_guest_tools_wait.py

```diff
@@ -69,6 +69,12 @@
      - Max duration of the waiting period (seconds).
      default: 500
      type: int
+   datacenter:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.